example-todomvc/build.sbt

```diff
@@ -0,0 +1,15 @@
+enablePlugins(ScalaJSPlugin, ScalaJSBundlerPlugin)
```

Comment:
Before merging I would like to merge that with the main `build.sbt` to have it at least compiled by the CI.